tools/maint/gen_sig_info.py

```diff
@@ -172,6 +172,8 @@ def ignore_symbol(s, cxx):
     return True
   if s.startswith('emscripten_gl') or s.startswith('emscripten_alc'):
     return True
+  if (s.endswith('_calling_thread') or s.endswith('_main_thread')) and not s.startswith('emscripten_asm_const'):
```

Comment:
Maybe add a comment here? I can't tell what this change is doing.

Reply:
Done

Reply:
I've updated this PR such that this change is no longer needed, so I split the doc changes out into a separate PR.